server/database/queries/getNumberPledgesCategory.js

```diff
@@ -0,0 +1,10 @@
+const dbConnection = require('./../db_connection')
+
+const getNumberPledgesCategory = (categoryName) => {
+  return dbConnection
+    .query('select * from pledge inner join category on pledge.category_id =category.category_id ')
```

Comment:
Capitalize the query statements like `SELECT, FROM, ...`

Reply:
done